content/templates.html

```diff
@@ -0,0 +1,120 @@
+<!DOCTYPE html>
+
+<meta charset=utf-8>
+<title>Styleguide - Neomad</title>
+<meta name=viewport content="width=device-width, initial-scale=1, maximum-scale=1">
+
+<link rel=stylesheet href=dist/styleguide.css>
+
+<script src=dist/assets/js/utils.js></script>
+
+<header class=main>
+  <h1>
+    <a href=index.html>Neomad - Styleguide</a>
+  </h1>
+  <nav>
+    <a href=atoms.html>Atoms</a>
+    <a href=molecules.html>Molecules</a>
+		<a href=components.html>Components</a>
+		<a href=templates.html>Templates</a>
+  </nav>
+</header>
+
+<main class="st-main">
+  <div class=st-title>Templates</div>
+
+  <section class=st-section>
+    <div class=st-title>Article list</div>
+
+    <div class="st-container st-full">
+			<section class=article-list>
```

Comment:
what happened to the indentation? I guess your editor is using tabs again and this one is doubled up.

Reply:
oupsy, addressed with 61889cc5efef33201d1de7aa852465c672477af0